Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/submissions/use-submissions-details.tsx

```diff
@@ -203,19 +159,18 @@ const SubmissionsDetailsProvider = ({ children }: ISubmissionsDetailsProviderPro
         [],
     );
 
-    useEffect(
-        () => {
-            if (isNil(apiSubmissionDetails)) {
-                return;
-            }
+    useEffect(() => {
+        if (isNil(apiSubmissionDetails)) {
```

Comment:
Make it something like this:
useEffect(
() => {
if (isNil(apiSubmissionDetails)) {
   return;
}
}